### PR TITLE
ADAreactor

### PR DESCRIPTION
**Pull request by coincooking**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -4425,5 +4425,20 @@
...
+   "poolId": "2fd4c1782db3cb6a97f910ca2c6f15914e2c45f75f1ca1cc168e319c",
+   "social": {
+     "website": "https://adareactor.com/",
+     "twitter": "ADAreactor",
+     "reddit": "ADAreactor",
+     "facebook": "",
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "adareactor_chat",
```
Comment (Bez625) [inline registry.json:4440]: Please change this to be a TG handle of an individual and not a group or channel

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (Bez625) [reply registry.json:4440]: Please change this to be a TG handle of an individual and not a group or channel

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by coincooking** [comment]
Hello.
Please tell me how I can do that. Thank you.


вс, 27 февр. 2022 г., 16:27 Chris Berry ***@***.***>:

**Comment by Bez625** [comment]
For your personal telegram account go to settings -> edit profile and you will see a screenshot like below. The Username is your telegram handle. Please provide this.

Screenshot from 2022-03-06 09-58-04

**Comment by Bez625** [comment]
Closing PR as no update in last month. Please raise another PR if you wish to join the alliance still

**Pull request closed without merging**